src/components/DocumentationTopic/Relationships.vue

```diff
@@ -44,6 +45,10 @@ export default {
       type: Array,
       required: true,
     },
+    enableMinimized: {
```

Comment:
I don't think you need to change this, but in the future it might be worth considering provide/inject for situations like this where you're adding props to a component solely for the purpose of passing it down to further nested children.